app.js

```diff
@@ -1,8 +1,29 @@
 // import functions and grab DOM elements
+import { plants } from './plants-data.js';
+import { renderPlant } from './render-plant.js';
+import { frogs } from './frogs-data.js';
+import { renderFrog } from './render-frog.js';
 
-// initialize global state
+const frogsList = document.getElementById('frogs-list');
+const plantsList = document.getElementById('plants-list');
 
-// set event listeners 
-  // get user input
-  // use user input to update state 
-  // update DOM to reflect the new state
+function displayPlants() {
+    plantsList.innerHTML = '';
+
+    for (let plant of plants) {
+        const plantEl = renderPlant(plant);
+        plantsList.append(plantEl);
+    }
```

Comment:
looping over array to create an element for each object and appending to parent container